pkg/support/dump_test.go

```diff
@@ -15,21 +15,129 @@
 package support
 
 import (
+	"fmt"
+	"path/filepath"
+	"strings"
 	"testing"
+	"time"
 
+	"github.com/spf13/afero"
 	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"k8s.io/utils/exec"
+	exectesting "k8s.io/utils/exec/testing"
+
+	"antrea.io/antrea/pkg/util/runtime"
 )
 
-func TestParseLogDate(t *testing.T) {
-	data := "I0817 06:55:10.804384       1 shared_informer.go:270] caches populated"
-	ts, err := parseTimeFromLogLine(data, "2021", "antrea-agent")
-	assert.Nil(t, err)
-	assert.Equal(t, ts.String(), "2021-08-17 06:55:10 +0000 UTC")
+type testExecuter struct {
+	exectesting.FakeExec
+}
 
-	data = "2021-06-01T09:30:43.823Z|00004|memory|INFO|cells:299 monitors:2 sessions:2"
-	ts, err = parseTimeFromLogLine(data, "2021", "ovs")
-	assert.Nil(t, err)
-	assert.Equal(t, ts.String(), "2021-06-01 09:30:43 +0000 UTC")
+// Command func is to track correct commands are executed
+func (te *testExecuter) Command(cmd string, args ...string) exec.Cmd {
```

Comment:
In this function, you are "faking" the command implementation, so that the command outputs the command name + args which which it was invoked. With that mechanism, you could verify that the right commands are executed when you invoke a function under test. However, I don't see that capability leveraged anywhere in your unit tests (at least, not any more, I seem to remember that an earlier version of this PR was using this).